pss/trojan/message.go

```diff
@@ -108,6 +113,20 @@ func (m *Message) Wrap(targets [][]byte) (chunk.Chunk, error) {
 	return chunk, nil
 }
 
+// Unwrap creates a new trojan message from the given chunk payload
+// this function assumes the chunk has been validated as a content-addressed chunk
+// it will return the resulting message if the unwrapping is successful, and an error otherwise
+func Unwrap(c chunk.Chunk) (*Message, error) {
```

Comment:
how about just returning the message.payload

Reply:
after unwrapping the chunk, we need its topic to find the corresponding handler func